src/test/java/org/folio/rest/impl/AccountsRefundAPITests.java

```diff
@@ -91,32 +112,36 @@ public void beforeEach() {
 
   @Test
   public void fullRefundOfClosedAccountWithPayment() {
-    double initialAmount = 6.0;
-    double payAmount = 5.0;
-    double transferAmount = 0.0;
-    double waiveAmount = 1.0;
-    double refundAmount = 5.0;
+    MonetaryValue initialAmount = new MonetaryValue(6.0);
+    MonetaryValue payAmount = new MonetaryValue(5.0);
+    MonetaryValue transferAmount = new MonetaryValue(0.0);
+    MonetaryValue waiveAmount = new MonetaryValue(1.0);
+    MonetaryValue refundAmount = new MonetaryValue(5.0);
+
+    MonetaryValue amountMinus5 = new MonetaryValue(-5.0);
```

Comment:
What is the actual meaning of this -5 amount? If it's just negative refund amount, maybe we can use refundAmount instead?